docs/howtos/vpn.md

```diff
@@ -1,16 +1,14 @@
 # Configure a VPN
 
-Podenv enables network to be shared between multiple environments.
-In this how-to guide we'll see how to use podenv to start openvpn and how
-to start further environment inside the new network.
+In this how-to guide we'll see how to use podenv to run openvpn.
 
 ## Create an openvpn container
 
```

Comment:
It is worth mentioning that "openvpn" application is already provided by the podenv HUB